test/test-server/kafka_rest_router.go

```diff
@@ -151,7 +151,7 @@ func handleKafkaRPTopics(t *testing.T) http.HandlerFunc {
 				"kind": "KafkaTopicList",
 				"metadata": {"self": "http://localhost:9391/v3/clusters/cluster-1/topics","next": null},
 				"data": [
-				  {
+					{
```

Comment:
Well this is annoying, but I'll leave it in because this is the go fmt'd output I got after I manually undid my changes here.

Reply:
I'll make a note to JSON-encode this in a separate PR